.changes/unreleased/tinymce-TINY-10971-2024-05-24.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Added
+body: Added a possibility to define for and id attributes. Label component has an optional
+  for spec parameter. Input, Colorinput, Checkbox, SelectBox and Textarea have an optional id spec parameter.
```

Maybe better not sure:

```suggestion
body: Added support for setting custom `for` attributes on label dialog components and an custom `id` attributes on the Input, Colorinput, Checkbox, SelectBox and Textarea components.
```